http-agent/src/test/java/org/motechproject/http/agent/listener/HttpClientEventListenerTest.java

```diff
@@ -37,27 +43,41 @@
     HttpComponentsClientHttpRequestFactory httpComponentsClientHttpRequestFactory;
     @Mock
     SettingsFacade settings;
+    @Mock
+    HTTPActionService httpActionService;
+    @Mock
+    ResponseEntity<?> responseEntity;
+    @Mock
+    HTTPActionRecord httpActionRecord;
 
-    private HttpClientEventListener httpClientEventListener;
 
+    private HttpClientEventListener httpClientEventListener;
     private String url;
     private String data;
     private Map headers;
     private Credentials credentials;
+    private HttpEntity httpEntity;

```

Comment:
Any reason httpEntity and request are class fields? It seems like they can be method variables.